extensions/wikia/AdEngine/js/AdTracker.js

```diff
@@ -73,6 +78,29 @@ define('ext.wikia.adEngine.adTracker', ['ext.wikia.adEngine.utils.timeBuckets','
 	}
 
 	/**
+	 * A generic function to track an ad-related event and its timing in DataWarehouse
+	 *
+	 * @param {object} data - data to track as JS object (will be converted to URL-like query-string)
+	 * @param {string} eventName 
```

Comment:
What do you think of adding a little bit more description to the 2nd param, even if it's really self-explanatory ;) I'm thinking of leaving an info here it's a part of track URL which can be also called `stream name` :japanese_ogre: